lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -48,7 +48,7 @@ sub _type { 'artist' }
 
 sub _columns
 {
-    return 'artist.id, artist.gid, artist.name, artist.sort_name, ' .
+    return 'artist.id, artist.gid, artist.name COLLATE musicbrainz, artist.sort_name, ' .
```

Comment:
If we're already doing this, maybe we should also `COLLATE` `sort_name` here and have a proper look at how we use `COLLATE` elsewhere, but I guess that might be a lot of extra work to do it properly? If you think it's of use but not right now, I can make a ticket I guess. In any case, we have plenty of cases where we order by collated sort name - wondering if we shouldn't be doing that for all artist queries where we use name now even?

Reply:
I think it makes sense to include here, but would prefer looking at it later for now.  But the nice thing is that we can add `COLLATE` here without breaking any `COLLATE` expressions elsewhere in the queries.  At worst we just have some useless expressions lying around.

Reply:
(To be clear, I changed it to `artist.sort_name COLLATE musicbrainz` here, just haven't properly looked into our use of `COLLATE` elsewhere.)

Reply:
We can look into this further later.